osu.Game/Overlays/Mods/EditPresetPopover.cs

```diff
@@ -58,18 +59,30 @@ public EditPresetPopover(ModPresetPanel modPresetPanel)
                         Label = CommonStrings.Description,
                         TabbableContentContainer = this
                     },
-                    useCurrentSwitch = new LabelledSwitchButton
+                    new FillFlowContainer
                     {
                         Anchor = Anchor.TopCentre,
                         Origin = Anchor.TopCentre,
-                        Label = "Use Current Mod select",
-                    },
-                    createButton = new ShearedButton
-                    {
-                        Anchor = Anchor.TopCentre,
-                        Origin = Anchor.TopCentre,
-                        Text = CommonStrings.MenuBarEdit,
-                        Action = tryEditPreset
+                        Direction = FillDirection.Horizontal,
+                        AutoSizeAxes = Axes.Y,
+                        Spacing = new Vector2(7),
+                        Children = new Drawable[]
+                        {
+                            useCurrentModButton = new ShearedButton
+                            {
+                                Anchor = Anchor.TopCentre,
+                                Origin = Anchor.TopCentre,
+                                Text = "Use Current Mods",
```

Comment:
Probably reads better as "overwrite with currently selected mods".

I also think this would work well as a (dangerous?) context menu item.